src/Meshes/trailMesh.ts

```diff
@@ -26,17 +26,17 @@ export class TrailMesh extends Mesh {
      * @param name The value used by scene.getMeshByName() to do a lookup.
      * @param generator The mesh to generate a trail.
      * @param scene The scene to add this mesh to.
-     * @param diameter Diameter of trailing mesh. Default is 1.
+     * @param dimensions Dimensions of trailing mesh. Default is [1,1].
      * @param length Length of trailing mesh. Default is 60.
      * @param autoStart Automatically start trailing mesh. Default true.
      */
-    constructor(name: string, generator: AbstractMesh, scene: Scene, diameter: number = 1, length: number = 60, autoStart: boolean = true) {
+    constructor(name: string, generator: AbstractMesh, scene: Scene, dimensions: Array<number> = [1, 1], length: number = 60, autoStart: boolean = true) {

```

Comment:
Would it be more useful to use Vector2 instead here?

Reply:
After some testing and adjustments, the first number of the array doesn't really change much anyway, so perhaps I should switch it back to diameter or call it width or height.

Reply:
Makes sense, width sounds good to me, and you would always set the first number to 0 and this parameter would control the second number?

Reply:
I have set it to be width. However, the trail is static to that plane of the mesh. Changing generator mesh scaling from y to x results in:
https://brave-gates-959b79.netlify.com/
so I need to figure out a way to construct the trail independent of the orientation of the generator mesh. This example has _sectionPolygonPointsCount set to 3. Maybe have option to set initial orientation x,y,or z and then use 2 vertices with the highest and lowest values of that plane and the generator mesh center for the 3rd point
if that makes sense?

Reply:
Sounds reasonable to me.